graylog2-server/src/main/java/org/graylog2/plugin/journal/RawMessage.java

```diff
@@ -129,6 +129,10 @@ public RawMessage(JournalMessage journalMessage, Object messageQueueId) {
 
     @Nullable
     public static RawMessage decode(final byte[] buffer, final Object messageQueueId) {
+        if (buffer == null) {
+            log.error("Cannot read <null> message from journal, ignoring this message.");
+            return null;
+        }
```

Comment:
Is this change related to the PR?

Reply:
Kind of, the related cloud plugin one uses a null Message as a Kafka canary.
We might want to change the canary.
But avoiding a NPE here is probably a good idea anyway..